CHANGELOG.md

```diff
@@ -1,13 +1,25 @@
 # SingularityCE Changelog
 
-## Changes Since Last Release
+## 4.0.0-rc.2 (Release Candidate 2) \[2023-09-05\]
+
+### OCI-mode
+
+- Use a temporary sandbox directory for the OCI bundle when it is not possible
```

Comment:
Should these additions exist _alongside_ lines 39 and 193, respectively - or instead of them?